docs/helm-chart-checks.md

```diff
@@ -131,18 +131,34 @@ This section provides help on the basic usage of Helm chart checks with the podm
 
 ### Cluster Config
 
-Configuration of the `chart-testing` check can be achieved through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
+Configuration of the `chart-testing` check can be achieved in one of two ways: 
```

Comment:
Suggestion: Replace this sentence with the following one as the 1st step:

1. You can configure the `chart-testing` check by performing one of the following steps:

- option 1 
- option 2

Reply:
Done.